content/en/docs/installation/uninstall/kubernetes.md

```diff
@@ -45,9 +45,13 @@ installation process, *in reverse*, using the delete command on both `kubectl`
 and `helm`.
 
 Firstly, delete the cert-manager installation using `helm`. Ensure the
-`--purge` flag is applied.
+`--purge` flag is applied if you are using Helm 2.
 
 ```bash
+# Helm 3
+$ helm delete cert-manager
```

Comment:
May also be worth including the `--namespace` arg too?

Reply:
Good idea!

Reply:
Added